### PR TITLE
[IMP] payment_acquirers: revamp online payment providers' documentation

### PR DESCRIPTION
**Pull request by Demesmaeker**
task-2654702

### CODE REVIEW

**Comment by Feyensv** [comment]
The commit can be an ADD since you are adding content.
Also, do not target payment_adyen since we're in the doc, not in odoo/odoo ;)
Could be something like `[ADD] general/payment_acquirers: ...`

File: content/applications/general/payment_acquirers/adyen.rst [hunk 1]
```diff
@@ -0,0 +1,108 @@
+=====
+Adyen
+=====
+`Adyen <https://www.adyen.com/>`__ is a Dutch-based company that allows businesses to
```
Comment (Feyensv) [inline content/applications/general/payment_acquirers/adyen.rst:4]:
```suggestion

`Adyen <https://www.adyen.com/>`__ is a Dutch-based company that allows businesses to
```

File: content/applications/general/payment_acquirers/adyen.rst [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+accept **online** and **point-of-sale** payments.
```
Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:5]: that offers several online payment solutions/methods/whatever -> don't mention pos

File: content/applications/general/payment_acquirers/adyen.rst [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+- Recurring API URL
+
+To retrieve the API Key, log into your Adyen account, go to :menuselection:`Account --> API
+Credentials --> Your username --> Authentication`, and generate your **API Key**.
+
+.. image:: media/adyen_api_key.png
+   :align: center
+   :alt: Generate a new API Key.
+
+To retrieve the HMAC Key, TODO
```
Comment (Feyensv) [inline content/applications/general/payment_acquirers/adyen.rst:44]: TODO :D

File: content/applications/general/payment_acquirers/adyen.rst [hunk 4]
```diff
@@ -0,0 +1,108 @@
+=====
+Adyen
+=====
+`Adyen <https://www.adyen.com/>`__ is a Dutch-based company that allows businesses to
+accept **online** and **point-of-sale** payments.
+
+.. image:: media/adyen_logo.png
+   :align: center
+   :alt: Adyen logo
```
Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:9]:
```suggestion
```

File: content/applications/general/payment_acquirers/adyen.rst [hunk 5]
```diff
@@ -0,0 +1,108 @@
+=====
+Adyen
+=====
```
Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:3]:
```suggestion
=====

```

File: content/applications/general/payment_acquirers/adyen.rst [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+.. image:: media/adyen_capture_delay.png
+   :align: center
+   :alt: Capture Delay settings in Adyen
+
+.. warning::
+   If you set the Capture Amount Manually on Odoo but not on Adyen, Odoo won't be able to see it,
+   your request for authorization will result in an authorization and immediate capture.
+
+
```
Comment (Feyensv) [inline content/applications/general/payment_acquirers/adyen.rst:89]: one empty line can be removed :D

File: content/applications/general/payment_acquirers/adyen.rst [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+`Adyen <https://www.adyen.com/>`__ is a Dutch-based company that allows businesses to
+accept **online** and **point-of-sale** payments.
+
+.. image:: media/adyen_logo.png
+   :align: center
+   :alt: Adyen logo
+
+This Payment Acquirer offers additional options that are not available for other :doc:`Payment
+Acquirers <../payment_acquirers>`, such as the ability to refund partially a payment directly from
+Odoo, to capture the payment after the delivery, or to link a point of sale's terminal.
```
Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:13]:
```suggestion
```

File: content/applications/general/payment_acquirers/adyen.rst [hunk 8]
```diff
@@ -0,0 +1,108 @@
...
+
+Payment Acquirer Configuration
```
Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:15]:
```suggestion
Configuration
```

File: content/applications/general/payment_acquirers/adyen.rst [hunk 9]
```diff
@@ -0,0 +1,108 @@
...
+
+.. warning::
+   If you set the Capture Amount Manually on Odoo but not on Adyen, Odoo won't be able to see it,
+   your request for authorization will result in an authorization and immediate capture.
+
+
+To capture the payment, go to the related Sales Order and click on *Capture Transaction*. If the
+order is canceled, you can click on *Void Transaction* to unlock the funds from the customer's card.
+
+.. image:: media/authorize-capture.png
```
Comment (Feyensv) [inline content/applications/general/payment_acquirers/adyen.rst:93]: rename `authorize-capture` file to `capture`, to make clear it's not only used by authorize page.

File: content/applications/general/payment_acquirers/adyen.rst [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+- **Payment API URL** from the Authorize (`https://payment_url/Payment/`)
+- **Recurring API URL** from the Recurring (`https://recurring_url/Recurring/`)
+
+.. image:: media/adyen_api_urls.png
+   :align: center
+   :alt: Get the links for the different API.
+
+.. note::
+   Please don't write the end of these endpoints, but let the trailing Forward Slash `/` as shown in
+   the exemples.
```
Comment (Feyensv) [inline content/applications/general/payment_acquirers/adyen.rst:59]:
```suggestion
   the examples.
```

File: content/applications/general/payment_acquirers/adyen.rst [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+   :alt: Adyen logo
+
+This Payment Acquirer offers additional options that are not available for other :doc:`Payment
+Acquirers <../payment_acquirers>`, such as the ability to refund partially a payment directly from
+Odoo, to capture the payment after the delivery, or to link a point of sale's terminal.
+
+Payment Acquirer Configuration
+==============================
+
+If not done yet, `contact Adyen <https://www.adyen.com/contact/sales>`__ to get your live account.
```
Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:18]: To proceed with... you need an Adyen account in the live environment (rephrase this).

Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:18]:
```suggestion
If not done yet, `contact Adyen <https://www.adyen.com/contact/sales>`_ to get your live account.
```

File: content/applications/general/payment_acquirers/adyen.rst [hunk 12]
```diff
@@ -0,0 +1,108 @@
...
+
+.. seealso::
+
+   - `Get started with Adyen
+     <https://docs.adyen.com/get-started-with-adyen>`__
```
Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:23]:
```suggestion
```

File: content/applications/general/payment_acquirers/adyen.rst [hunk 13]
```diff
@@ -0,0 +1,108 @@
...
+
+Credential
```
Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:25]:
```suggestion
Credentials
```

File: content/applications/general/payment_acquirers/adyen.rst [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+----------
+
+Odoo needs your **API Credentials** to connect with your Adyen account, which comprise:
+
+- Merchant Account
+- API Key
+- HMAC Key
+- Checkout API URL
+- Payment API URL
+- Recurring API URL
```
Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:35]: description

File: content/applications/general/payment_acquirers/adyen.rst [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+--------------------------------------
+
+The **Capture Amount Manually** field is under the **Configuration** tab. If enabled in Odoo and in
+Adyen, the funds are reserved on the customer's card, but not charged yet.
+
+.. image:: media/adyen_capture_manually.png
+   :align: center
+   :alt: Adyen Configuration tab on Odoo
+
+On your Adyen Merchant Account, go to :menuselection:`Account --> Settings`, and set the **Capture
```
Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:78]: If you whish to

File: content/applications/general/payment_acquirers/adyen.rst [hunk 16]
```diff
@@ -0,0 +1,108 @@
...
+.. image:: media/adyen_capture_delay.png
+   :align: center
+   :alt: Capture Delay settings in Adyen
+
+.. warning::
+   If you set the Capture Amount Manually on Odoo but not on Adyen, Odoo won't be able to see it,
+   your request for authorization will result in an authorization and immediate capture.
+
+
+To capture the payment, go to the related Sales Order and click on *Capture Transaction*. If the
```
Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:90]:
```suggestion
```

File: content/applications/general/payment_acquirers/adyen.rst [hunk 17]
```diff
@@ -0,0 +1,108 @@
...
+.. image:: media/authorize-capture.png
+   :align: center
+   :alt: Hold the credit card payment until you capture or revoke it on Odoo
+
+.. warning::
+   After **7 days**, the authorization can be **revoked** by the customer if it's still not
+   captured.
+
+.. note::
+   With other payment acquirers, you can manage the capture in their own interfaces, not from Odoo.
```
Comment (AntoineVDV) [inline content/applications/general/payment_acquirers/adyen.rst:102]:
```suggestion
```

File: content/applications/general/payment_acquirers/adyen.rst (hunk 3 above)
Comment (Demesmaeker) [reply content/applications/general/payment_acquirers/adyen.rst:44]: I said it before we made the review!  =D

File: content/applications/general/payment_acquirers/adyen.rst [hunk 18]
```diff
@@ -0,0 +1,143 @@
...
+
+`Adyen <https://www.adyen.com/>`_ is a Dutch-based company that offers several online payment
+possibilities.
+
+Configuration
+=============
+
+If not done yet, `contact Adyen <https://www.adyen.com/contact/sales>`_ to get your live account.
+
+TODO (@vfe : Not yet done =P)
```
Comment (Feyensv) [inline content/applications/general/payment_acquirers/adyen.rst:14]: :eyes:

File: content/applications/general/payment_acquirers/adyen.rst (hunk 9 above)
Comment (Feyensv) [reply content/applications/general/payment_acquirers/adyen.rst]: Okay, I'm stupid, it's authorize for authorization, and not authorize for Authorize.net :man_facepalming:. (You can keep the old name :grinning:)

**Comment by AntoineVDV** [comment]
@robodoo r+

**Pull request closed without merging**